docs/guides/graphics_tutorial.rst

```diff
@@ -0,0 +1,279 @@
+
+.. image:: images/graphics_tutorial/change_model_scale.png
+	:alt: Change Model Scale and LOD Distance
+	:align: center
+
+|
+
+We need to create an empty scene and add a light component to it to add light.
+
+***************
+Light Component
+***************
+
+To add light, we now create an empty scene.
+
+1) Name the scene.
+2) Add transform and directional light components.
+
+.. image:: images/graphics_tutorial/add_sun_scene.png
+	:alt: Directional Light Added
+	:align: center
+.. image:: images/graphics_tutorial/edit_sun_components.png
+	:alt: Added Components
+	:align: center
+
+|
+
+To move freely, we can change our camera mode to Editor Camera. This allows us to move freely.
+
+*************
+Editor Camera
+*************
+
+To have complete control of movement, you can use an editor camera.
+
+1) Click the image icon at the top left of the viewport.
+2) Open dropdown of camera.
+3) Select editor camera.
+
+.. image:: images/graphics_tutorial/change_to_editor_camera.png
+	:alt: Change To Editor Camera
+	:align: center
+	
+To move, you have to hold the right mouse button and then use WASD space and shift keys to move. The cursor for direction. Space to move up and shift to move down.
+
+.. image:: images/graphics_tutorial/navigate_with_editor_camera.png
+	:alt: Navigate With Editor Camera
+	:align: center
+	
+
+
+***********
+Point Light
+***********
+
+A point light is helpful if you have a source of light, e.g. a candle, bulb etc. To add a point light, follow the given steps.
+
+1) Add an empty scene and give it a point light component.
+2) You can tweak its transformation value by either inputting it or dragging it left or right. 
+
+.. image:: images/graphics_tutorial/add_point_light_texture.png
+	:alt: Add Point Light
+	:align: center
+
+
+If you press 'q', a transform gizmo will appear on the object you have selected. You can adjust light location through it.
+For rotation and scaling gizmo, press 'w' and 'e', respectively.
+
+.. image:: images/graphics_tutorial/translate_point_light.png
+	:alt: Translate Point Light 1
+	:align: center
+.. image:: images/graphics_tutorial/translate_point_light_2.png
+	:alt: Translate Point Light 2
+	:align: center
+.. image:: images/graphics_tutorial/translate_point_light_3.png
+	:alt: Translate Point Light 3
+	:align: center
+
+
+*********************
+Overriding a material
+*********************
+
+To change the properties of one object without changing the original material, we can use overriding materials. To override a material:
+
+1) Create a new material by going to file -> Create Resource.
+2) Name the material and click create.
+
+.. image:: images/graphics_tutorial/create_basic_material.png
+	:alt: Create Basic Material
+	:align: center
+
+3) Go to the `inspector-> model component->Materials`.
+4) click on the folder icon on the corresponding overriding material.
```

Comment:
Again, tell the filepath of the material

Reply:
Again, it's visible in the screenshot

Reply:
Better to have it in text too. Makes the tutorial reading easy.

Reply:
done